src/views/collection-view/server.ts

```diff
@@ -165,11 +164,9 @@ export async function getCollectionPagePaths(): Promise<CollectionPagePath[]> {
 		 * @TODO once we implement the `theme` query option, remove the theme filtering
 		 * https://app.asana.com/0/1201903760348480/1201932088801131/f
 		 */
-		const isBetaProduct =
-			isProductSlug(productSlug) && getIsBetaProduct(productSlug)
 		const isCloud = collectionProductSlug == 'cloud'
 		const themeMatches = collectionProductSlug === collection.theme
-		const shouldBuildCollectionPath = isBetaProduct && (isCloud || themeMatches)
+		const shouldBuildCollectionPath = isCloud || themeMatches
```

Comment:
@kendallstrautman I might need your 👀 here on what the expected logic should be. Do we still need the theme check?

Reply:
Yes we do! This ensures we only build collections where the product is the main `theme` (in the future we could adjust this to reference `section`). cloud, waf and onboarding are the exceptions to this rule.